### ISSUE TITLE
max_rates_ph generates warning

### ISSUE THREAD

**Issue by ghost** [VanAndelInstitute/cardiocyte#3]
This is probably a symptom of badness:
 
```
Loading cardiocyte
Testing cardiocyte
v |  OK F W S | Context
v |   1       | Baseline Correction
v |  15   1   | Signal parameters [0.2 s]
------------------------------------------------------------------------------------------------------------------
test-parameters.R:70: warning: Max Rates per peak height can be calculated
longer object length is not a multiple of shorter object length
------------------------------------------------------------------------------------------------------------------
```

**Comment by ghost** [comment]
@JoshBauman  length of peak heights does not always equal length of velocities.  Maybe related to #2?

**Comment by ghost** [comment]
Fixed by improving peak detection in commit d7941f7
